tests/python/test_options.py

```diff
@@ -0,0 +1,61 @@
+import importlib.resources
+import os
+import mutationpp as mpp
+
+ENV_DATADIR = os.environ.get("MPP_DATA_DIRECTORY")
+PKG_DATADIR = os.fspath(importlib.resources.files(mpp) / "data")
+
+
+def test_options_datadir_default():
+    datadir = mpp.GlobalOptions.dataDirectory()
+    if ENV_DATADIR is not None:
```

Comment:
I think you want to probably use `unittest.patch.dict` to patch  `os.environ`  instead of relying on the real environment for the testing. 

You can give a look to `pytest-mock` for a slightly more integrated way,

Reply:
Done. Please give it a new look. There are two tests that still rely on the global environment to check for the import-time initial value of the data/working directories. IMHO, these test should be kept as they are. However, if you do not like them, I have not problem with removing them. The other tests now use pytest fixtures and os.environ patching. The test code looks now less convoluted.